R/calculate_recommended_interventions.R

```diff
@@ -1,123 +1,170 @@
 #' calculate_recommended_interventions
 #'
-#' @description Returns the LAGO recommended interventions for the next stage.
-#' The recommended interventions that aim to satisfy the outcome goal and/or the
+#' @description Calculates the LAGO recommended interventions for the next stage.
+#' The recommended interventions that aim to satisfy an outcome goal and/or a
 #' power goal.
 #'
-#' @param df data.frame, Input data
-#' @param outcome_name character, Outcome name
-#' @param outcome_type character, Outcome type, either "continuous" or "binary"
-#' @param glm_family character, Family for the GLM model
-#' For example: "binomial"
-#' For v1.1, we are only considering binomial family, and quasibinomial(link="logit")
-#' @param link character, the link option when glm_family is set to quasibinomial.
-#' It has to be one of the following:
-#' logit (default), probit, cauchit, log and cloglog.
-#' @param include_intercept boolean, Whether the intercept should be included in the
-#' fitted model.
-#' @param interventions_list character vector, Names of the intervention components
-#' For example: c("component1", "component2")
-#' @param center_characteristic_list character vector, Names of the center characteristics
-#' For example: c("characteristic1")
-#' @param center_characteristic_list_for_optimization numeric vector, Given values of
-#' the center characteristics
+#'
+#' ## required arguments:
+#' @param data data.frame: input data frame.
+#' @param outcome_name character: column name of the outcome of interest.
+#' @param outcome_type character: type of the outcome of interest,
+#' either "continuous" or "binary".
+#' @param intervention_components character vector: a vector of column names of the intervention
+#' components. For example: c("component1", "component2")
+#' @param intervention_lower_bounds numeric vector: lower bounds of the intervention components
+#' For example: for a two-component intervention package, lower bounds could be
+#' c(0,0).
+#' @param intervention_upper_bounds numeric vector: upper bounds of the intervention components
+#' For example: for a two-component intervention package, upper bounds could be
+#' c(10,20).
+#'
+#'
+#' ## optional arguments:
+#' @param glm_family character: GLM family of the intended outcome model.
+#' automatically determined if "auto":
```

Comment:
I'm slightly confused with the wording, does that mean if the user does not include this parameter in the model, it is automatically determined to be either gaussian or binomial? Or do they have to specify "auto"? I think you meant the first one... Maybe something along the line of "Default value without user specification is..."

Reply:
yeah its the first one, will change the wording to reflect that.